### PR TITLE
Updating atheros firmware and wlan interface ajustments

### PR DESCRIPTION
**Pull request by keomabrun**
The firmwares are built using the latest commit of this repo:
https://github.com/qca/open-ath9k-htc-firmware/commit/014ff6fde0fab968eb5b4365e83a93a2fff452c2
I used the latest commit and not the latest tag (1.4.0) as I was faced with build issues.

NOT TESTED YET

### CODE REVIEW

File: scripts/install [hunk 1]
```diff
@@ -248,6 +248,12 @@ sed 's|<TPL:NGINX_REALPATH>|/var/www/wifiadmin/|g' -i /etc/php5/fpm/pool.d/wifia
 # Fix sources
 sed "s|<TPL:NGINX_LOCATION>|${path_url}|g" -i /var/www/wifiadmin/config.php
 
+# Fix network interface
+grep -q -x -F 'auto wlan0' /etc/network/interfaces || echo 'auto wlan0' >> /etc/network/interfaces
```
Comment (alexAubin) [inline scripts/install]: (Can we also grep in `interfaces.d` just in case ? :wink: )

Comment (keomabrun) [reply scripts/install]: ok I'll do ! Thanks

**Comment by keomabrun** [comment]
Works well but requires a system restart for the firmware to be loaded (reloading with `modprobe` does not work).

**Comment by keomabrun** [comment]
We need to copy the firmware files only if the firmware is not already present on the system.

**Comment by keomabrun** [comment]
Looks like the ath9k_htc kernel option is already activated.
https://github.com/armbian/build/blob/master/config/kernel/linux-sun7i-default.config#L1633

**Pull request closed without merging**